app/services/host_aspects.rb

```diff
@@ -0,0 +1,11 @@
+module HostAspects
```

Comment:
If the plan is to use aspects not just with hosts, but also hostgroups (possibly something else?), it would be better not to use 'Host' in here.